src/parser/resolver/accessors/abstract-collector.ts

```diff
@@ -23,11 +23,12 @@ export abstract class AbstractCollector {
     ): Map<string, Accessor[]> {
         const publicAccessorsMap = new Map<string, Accessor[]>();
 
-        if (this.getAccessorsQuery()) {
```

Comment:
this is to avoid that the function `this.getAccessorQuery()` get called too often, since it has a huge complexity.